python/paddle/distributed/passes/gradient_merge.py

```diff
@@ -0,0 +1,349 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Please rename `gradient_merge.py` to `auto_parallel_gradient_merge.py` since this pass may not work for other codes.

Reply:
done